.travis.yml

```diff
@@ -94,6 +94,11 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_native_nowallet.sh"
 
+    - stage: test
+      name: 'ARM64 Android APK  [GOAL: build]'
+      env: >-
+        FILE_ENV="./ci/test/00_setup_env_android.sh"
```

Comment:
We no longer use travis, so you'll have to move those added lines over to cirrus

Reply:
Done